README.md

```diff
@@ -1 +1,104 @@
-# sample-gateway-mock-apis
\ No newline at end of file
+# Mock APIs via Spring Cloud Gateway
+
+Spring Cloud Gateway can be used to Mock APIs while still providing typical gateway proxying functionality. The ability to mock specific APIs can be extremely helpful during frontend or backend SDLC phases, when dependencies aren’t yet available.
+
+This service is built on top of [Spring Cloud Gateway](https://cloud.spring.io/spring-cloud-gateway/reference/html/). To mock endpoints, add property based configuration at [config-props](). Configuration can also be used to add proxy configuration if needed.
```

Comment:
Fix the dead config-props link or remove the link since there's a separate section below

Reply:
updated sentence